wp-content/themes/freshenergy/functions.php

```diff
@@ -77,6 +77,17 @@ function category_and_tag_archives( $wp_query ) {
 }
 
 
+/**
+ * oembed is a pain; this fixes fixed widths for responsive output
+ *
+ * Wraps embeds in a container div.
+ */
+function mwe_responsive_embed($html, $url, $attr, $post_ID) {
+	$return = '<div class="video-container">'.$html.'</div>'; return $return;
+}
+add_filter( 'embed_oembed_html', 'mwe_responsive_embed', 10, 4 );
```

Comment:
This takes 4 parameters, but only uses the `$html`. And [this comment](https://developer.wordpress.org/reference/hooks/embed_oembed_html/#comment-2414) says that the example there, which looks the same as this code, appears to affect *all* URLs, not just YouTube's.

So some questions:

- Can this be limited via $url or $attr or something to only affect YouTube embeds?
- How does this change affect embeds from sites other than YouTube? The full list of oembed providers is here: https://codex.wordpress.org/Embeds At minimum we'd want to check:
    - Instagram
    - Facebook
    - Twitter

And like @kaylima said, the prefix here should be changed to reflect that this isn't `mwen_`, but is `fe_`

Reply:
also update for MWE